Add 2023 Sponsor Run of Show (#1924)
writethedocs/www: docs/conf/portland/2023/sponsors/prospectus.rst

{% else %}

.. contents::
   :local:
   :depth: 1
   :backlinks: none

This Run of Show provides more context about the event and answers some common questions you may have.
Please let us know if there is any information missing that would be useful for you.

Quick tips
----------

* **Arrive early**: The conference venue opens at 8am, so we recommend arriving around this time to get the most interaction with attendees.

> * **Bring swag (especially stickers)**: We recommend bringing a variety of swag to give away. Stickers are by far the most popular item, so that would be our top priority.

Reviewer: ```suggestion
* **Bring swag (especially stickers)**: We recommend bringing a variety of swag to give away. Stickers are by far the most popular item for our attendees.
```
